compiler/rustc_trait_selection/src/traits/project.rs

```diff
@@ -164,18 +164,19 @@ pub(super) enum ProjectAndUnifyResult<'tcx> {
 /// If successful, this may result in additional obligations. Also returns
 /// the projection cache key used to track these additional obligations.
 ///
-/// ## Returns
+/// ## Returns (`ProjectAndUnifyResult`)
```

Comment:
Can you instead move that comment to the type definition of `ProjectAndUnifyResult`?

Reply:
Done.